prog/vm/nexus.rb

```diff
@@ -239,9 +239,6 @@ def before_run
         storage: vm.storage_secrets
       })
 
-      # Enable KVM access for VM user.
-      host.sshable.cmd("sudo usermod -a -G kvm #{q_vm}")
-
```

Comment:
reading this commit message, re: "I relocated it from `prep` to `create_unix_user` in the previous commit.
I removed it in this commit for a smoother deployment." 

Is the intermediate state (where we add the user to the KVM gruop twice) acceptable?

Reply:
We are moving adding the user to `kvm` group command to an earlier step. The second command is to ensure that all users are included in the `kvm` group if any strand has already passed the `create_unix_user` label but hasn't run `prep` label yet at the time of deployment.

Reply:
I run E2E pipeline on this middle step
https://github.com/ubicloud/ubicloud/actions/runs/10116708456